src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -426,6 +427,18 @@ protected DefaultDockerClient(final Builder builder) {
     this.headers = new HashMap<>(builder.headers());
   }
 
+  private ClientConfig updateProxy(ClientConfig config) {
+    final String proxyHost = System.getProperty("http.proxyHost");
+    if (proxyHost != null) {
+      config.property(ClientProperties.PROXY_URI, proxyHost + ":"
+              + System.getProperty("http.proxyPort", "8080"));
```

Comment:
if you have already checked that this property exists, is there any point in passing the second argument?